Mathlib/Combinatorics/Additive/PluenneckeRuzsa.lean

```diff
@@ -19,26 +19,30 @@ inequality.
 
 ## Main declarations
 
-* `Finset.ruzsa_triangle_inequality_sub_sub_sub`: The Ruzsa triangle inequality, difference version.
-* `Finset.ruzsa_triangle_inequality_add_add_add`: The Ruzsa triangle inequality, sum version.
-* `Finset.pluennecke_petridis_inequality_add`: The Plünnecke-Petridis inequality.
-* `Finset.pluennecke_ruzsa_inequality_nsmul_sub_nsmul_add`: The Plünnecke-Ruzsa inequality.
+* `Finset.card_sub_mul_le_card_sub_mul_card_sub`: Ruzsa's triangle inequality, difference version.
```

Comment:
The changest in this file seem to be undoing recent changes on `master`.

Reply:
Fixed.